runtime/usr/local/bin/start-server

```diff
@@ -4,10 +4,15 @@ if [[ "${SERVER}" == spawn-fcgi ]]; then
     echo "Starting with spawn-fcgi"
     # Save the environment to be able to restore it in the FCGI daemon (used in /usr/local/bin/mapserv_wrapper)
     ${GET_ENV} | sed -e 's/.\+/export "\0"/' > /tmp/init_env
-    /usr/bin/spawn-fcgi -p "${LIGHTTPD_FASTCGI_PORT}" -n -- /usr/local/bin/mapserv_wrapper
+    if [[ "${LIGHTTPD_FASTCGI_SOCKET}" == "" ]]; then
+        exec /usr/bin/spawn-fcgi -p "${LIGHTTPD_FASTCGI_PORT}" -n -- /usr/local/bin/mapserv_wrapper
+    else
+        exec /usr/bin/spawn-fcgi -s "${LIGHTTPD_FASTCGI_SOCKET}" -n -- /usr/local/bin/mapserv_wrapper
+    fi
 else
     if [[ "${SERVER}" == lighttpd ]]; then
         echo "Starting lighttpd"
+        lighttpd -t -f "${LIGHTTPD_CONF}"
```

Comment:
I recommend `-tt` instead of `-t`.  Please run `lighttpd -h` to see the difference.